src/components/organisms/StartProjectPane/RecentProject.tsx

```diff
@@ -0,0 +1,73 @@
+import React, {FC, useState} from 'react';
+
+import Tooltip from 'antd/lib/tooltip';
+
+import {PushpinFilled, PushpinOutlined} from '@ant-design/icons';
+
+import {DateTime} from 'luxon';
+
+import {Project} from '@models/appconfig';
+
+import * as S from './RecentProject.styled';
+
+interface RecentProjectType {
+  project: Project;
+  isActive: boolean;
+  onProjectItemClick?: Function;
+  onPinChange?: Function;
```

Comment:
Is it difficult to properly type these functions? I notice they pass parameters.

Reply:
checked ts documentation for it but I couldn't find a proper way to do it with parameters